examples/gen_test_data/gen_test_data/utils/components.py

```diff
@@ -0,0 +1,51 @@
+import json
+import os
+from pathlib import Path
+
+from mldesigner import Input, Output, command_component
+from common import split_document, clean_data_and_save
+from constants import ENVIRONMENT_DICT_FIXED_VERSION
+
+
+@command_component(
+    name="document_split",
+    version="1.0.4",
```

Comment:
Why need this version? Do we need to change the version if we change the split document logic?

Reply:
To delete.